static/app/components/onboarding/gettingStartedDoc/replayConfigToggle.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled';
+
+import Switch from 'sentry/components/switchButton';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+function ReplayConfigToggle({
+  maskToggle,
+  onMaskToggle,
+  blockToggle,
+  onBlockToggle,
+}: {
+  blockToggle: boolean;
+  maskToggle: boolean;
+  onBlockToggle: () => void;
+  onMaskToggle: () => void;
+}) {
+  return (
+    <SwitchWrapper>
+      <SwitchItem>
+        {t('Mask All Text')}
+        <Switch
+          id={t('Mask All Text')}
+          toggle={onMaskToggle}
+          size="lg"
+          isActive={maskToggle}
+        />
+      </SwitchItem>
+      <SwitchItem>
+        {t('Block All Media')}
+        <Switch
+          id={t('Block All Media')}
+          toggle={onBlockToggle}
+          size="lg"
+          isActive={blockToggle}
+        />
+      </SwitchItem>
+    </SwitchWrapper>
+  );
+}
+
+const SwitchItem = styled('div')`
```

Comment:
a11y!

```suggestion
const SwitchItem = styled('label')`
```

That html tag, paired with prop `<SwitchItem htmlFor="id-of-the-switch" />` would make the label clickable too. The `id=` prop of the `<Switch/>` doesn't need to be translated, or in sentence form either.